### PR TITLE
Task 1/email address spec

### PR DESCRIPTION
**Pull request by chandur626**

### CODE REVIEW

File: cogs/deadline.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# TODO deadline reminder for all students
 # Copyright (c) 2021 War-Keeper
-# This functionality provides various methods to manage reminders (in the form of creation, retrieval, updation and deletion)
-# A user can set up a reminder, check what is due this week or what is due today. He/She can also check all the due homeworks based on hte coursename.
-# A user can also update or delete a reminder if needed.
+"""
+This functionality provides various methods to manage reminders (in the form of creation, retrieval,
```
Comment (nirajlavani) [inline cogs/deadline.py:3]: Does pylint require an indent here or no? If not, the file looks good 👍

**Comment by chandur626** [comment]
This pr consists of refactoring changes for email-address-spec.py deadline.py and voting.py files, making them comply with pylint recommendations.

**Review by HariniBharata — APPROVED** [review]
Approved.

**Review by 05sandesh — APPROVED** [review]

File: cogs/deadline.py (hunk 1 above)
Comment (chandur626) [reply cogs/deadline.py:3]: Not necessarily for any docstring but it is okay to not have the comment indented for file description.

**Review by nirajlavani — APPROVED** [review]

**Pull request merged**